examples/custom_search_method/asha_search_method/experiment_files/config.yaml

```diff
@@ -0,0 +1,9 @@
+name: mnist-custom-search-experiment
+data:
+  url: https://s3-us-west-2.amazonaws.com/determined-ai-test-data/pytorch_mnist.tar.gz
+searcher:
+  name: custom
+  metric: validation_loss
+  smaller_is_better: true
+  unit: batches
```

Comment:
what is units used for behind the scenes for custom searcher?  is it required?

Reply:
Units can be either epochs or batches, and yes, specifying these is required. We introduced it since (1) prev default behavior was using epoch if max length was not specified (I think this is not true anymore); and (2) validateAfter needs to know the unit (epoch or batches) (or rather master needs to know in which units validateAfter is talking).

Relevant thread is here: https://determined-ai.slack.com/archives/C03M8MBFK1N/p1660167154095449